leetcode/LinkedListTests/LinkedList_206.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace LinkedListTests;
+
+[TestFixture]
+class LinkedList_206
+{
+    private class SolutionRecursion {
+        public ListNode? ReverseList(ListNode? head) {
+            if(head is null || head.next is null) { 
+                return head; 
+            }
+            var last = ReverseList(head.next);
```

Comment:
Before this line:
A->B->........->Z->null
After this line we get:
Z->..........B
A->B->null